test/test_distributions.py

```diff
@@ -298,6 +302,38 @@ def test_bernoulli_3d(self):
                          (2, 5, 2, 3, 5))
         self.assertEqual(Bernoulli(p).sample_n(2).size(), (2, 2, 3, 5))
 
+    def test_binomial(self):
+        total_count = 10
+        p = Variable(torch.Tensor([0.1, 0.2, 0.3]), requires_grad=True)
+        self._gradcheck_log_prob(lambda p: Multinomial(total_count, p), [p])
+        self._gradcheck_log_prob(lambda p: Multinomial(total_count, None, p.log()), [p])
+        self.assertRaises(NotImplementedError, Multinomial(10, p).rsample)
+        self.assertRaises(NotImplementedError, Multinomial(10, p).entropy)
+
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
+    def test_binomial_log_prob(self):
+        total_count = 10
+        probs = Variable(torch.Tensor([0.1, 0.2, 0.3]), requires_grad=True)
```

Comment:
Could you add some more test data to provide more assurance that `log_prob` is correct?